src/components/Toolbar/Index.tsx

```diff
@@ -0,0 +1,92 @@
+import {
+    StyleSheet,
+    ToolbarAndroid,
+    ToolbarAndroidAction
+} from 'react-native';
+import * as React from 'react';
+import icons, {
+    images
+} from '../../assets/index';
+
+/*Only one button is permitted on the left
+it must either be a menu button which has a 'burger' icon
+or a back button which has an arrow icon*/
+export enum LeftButtonTypes {
+    menu = "menu",
+    back = "back"
+}
+/*Multiple buttons can be added to the right side of the toolbar
+if an icon isn't specified the title will be displayed in its place*/
+interface RightButtonType {
+    title: string;
+    icon ? : any;
+    action: () => void;
+}
+/*This specifies the toolbar properties we intend to allow the pages to control
+currently the title, left button type (menu or back) and right buttons can be specified*/
```

Comment:
I don't think it is necessary to explain the available options (the name of the variables should be named well enough that they are self-explanatory)

Reply:
removed the bottom line